subprojects/docs/src/docs/userguide/developing-plugins/implementing_gradle_plugins.adoc

```diff
@@ -9,7 +9,7 @@ This section assumes you have:
 
 - Basic understanding of software engineering practices
 - Knowledge of Gradle fundamentals like project organization, task creation and configuration as well as the Gradle build lifecycle
-- Working knowledge in writing Java code
+- Working knowledge in writing Java code (if you're writing your plugins in Kotlin make sure to check the <<writing_in_kotlin.adoc, Writing your plugins in Kotlin>> page in addition to this page)
```

Comment:
We should not just sneak this link in here, but have a dedicated section explaining that:
* plugins can be written in various JVM languages
* that Kotlin is one of those options and has dedicated documentation now
* that Java being used in this page just as an example